airbyte-integrations/connectors/source-exchange-rates/build.gradle

```diff
@@ -0,0 +1,29 @@
+plugins {
+    id 'airbyte-python'
+    id 'airbyte-docker'
+    id 'airbyte-standard-source-test-file'
+}
+
+airbytePython {
+    moduleDirectory 'source_exchange_rates'
+}
+
+airbyteStandardSourceTestFile {
```

Comment:
Following https://docs.airbyte.io/contributing-to-airbyte/building-new-connector/testing-connectors, is this enough to correctly set tests up?

Reply:
yup!